### ISSUE TITLE
import/named works fail with destructuring with default values

### ISSUE THREAD

**Issue by bigslycat** [benmosher/eslint-plugin-import#718]
## `env.js`:

```javascript
...

export const {
  ENTRY = rc.entry || './index.js',

  PATH = rc.path || resolve('dist'),
  PUBLIC = rc.publicPath || '/',
  FILENAME = rc.filename || 'bundle.js',
  NODE_ENV = 'development',
} = process.env;
```

## `other.js`:

![2017-01-11 14-46-58](https://cloud.githubusercontent.com/assets/1121997/21847538/69a9b118-d80d-11e6-94b4-4c46c5d39966.png)

All works correctly without default values.

**Comment by ljharb** [comment]
can you provide the errors as reported on the command line?

**Comment by vikr01** [comment]
Running into the same issue:

```javascript
export const {
  foo: bar = 'baz',
} = {};
```
